### ISSUE TITLE
Challenge

### ISSUE THREAD

**Issue by egregors** [egregors/MoonVision-Challenge#1]
Fullstack/Platform Engineer:
============================

Please pick one that best suits you or reflects your interest the most. A robust yet simple approach will win us over!

1) Solve an open issue from the Dask project. 
We use dask (https://dask.org) to schedule data transformations and model predictions. You can choose from any sub-project under the github organization (https://github.com/dask).
a) Present the link to the PR.
b) Explain why you have chosen this issue and what you are particularly proud of in your PR.

2) Django REST endpoint for image classification.
We use Django as the central framework for our API and cloud UI (app.moonvision.io). You should demonstrate familiarity with modeling state and computation for pre-trained model inference.
Proposed endpoint: POST request with an image binary and a string signifying the type of pretrained model, return the string of the label the image was classified with
The models can be from the hub of TorchVision (https://pytorch.org/docs/stable/hub.html), TensorFlow (https://www.tensorflow.org/hub) or similar
Bonus: provide a Dockerfile that we can run easily 
Did you see any difficulty in this API design? What would you change?
Explain in 4 sentences what you would do next to scale out this solution
